www/src/views/builder/index.ejs

```diff
@@ -505,7 +531,6 @@
 				</div>
 			</div>
 		</div>
-	</div>
```

Comment:
should I be concerned that this closing tag just disappeared? :P

Reply:
Well if you aren't, I am. It seems to work just fine though? I'll investigate this a bit.

Reply:
this is a mystery. My first thought was that I had a hidden </dive> at the end of a line in the new table, nope. It seems to be working fine though.

Reply:
yea this needs to be put back. I counted the divs and there is one with a missing closing tag. Browsers will try and fix this on the fly, but we can't rely on that.